src/jira.js

```diff
@@ -70,7 +70,7 @@ export default class JiraApi {
    * @property {string} [username] - Specify a username for this tool to authenticate all
    * requests with.
    * @property {string} [password] - Specify a password for this tool to authenticate all
-   * requests with.
+   * requests with. Cloud users need to generate an API token fo this value.
```

Comment:
you have a typo: "fo" should be "for". Also it might not be a bad idea to link to the Jira article on API token: https://confluence.atlassian.com/cloud/api-tokens-938839638.html

Reply:
Thanks for the catch and suggestion.  I just committed your suggestions to the branch.